### PR TITLE
GET /photo returns a random camera, GET /photo.json returns all

### PR DESCRIPTION
**Pull request by sshirokov**
`GET /photo.json` returns JSON with URLs to photos from all attached cameras:

As in: (I only have one camera in this machine)

``` json
{
  "images": [
    "http://localhost:9393/snap_1_1357951597.jpg"
  ]
}
```
## Goals
- [x] Redirect to random camera on `GET /photo`
- [x] Return all photos in JSON
- [x] Hubot integration

cc/ @skalnik

### CODE REVIEW

File: server/app.rb [hunk 1]
```diff
@@ -28,6 +28,16 @@ class App < Sinatra::Base
     end
 
     helpers do
+      def imagesnap
+        imagesnap = File.join(APP_ROOT, 'vendor', 'imagesnap', 'imagesnap')
+      end
```
Comment (skalnik) [inline server/app.rb]: Why not just make this a constant?

**Comment by sshirokov** [comment]
@skalnik: Factored out the entire /photo\* route set to `app/photo.rb`, which does, in fact move imagesnap to a constant.

**Comment by skalnik** [comment]
:+1: once we have Hubot Integration

**Comment by skalnik** [comment]
Hubot integration happening! https://github.com/github/hubot-scripts/pull/751

**Pull request merged**